hyperdx-helm/values.yaml

```diff
@@ -0,0 +1,62 @@
+fullnameOverride: "hdx-oss"
+
+global:
+  imageRegistry: ""
+  imagePullSecrets: []
+
+hyperdx:
+  apiKey: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+  apiPort: 8000
+  appPort: 3000
+  appUrl: "http://localhost"
+  logLevel: "info"
+  usageStatsEnabled: true
+
+mongodb:
+  port: 27017
+
+redis:
+  port: 6379
+
+clickhouse:
+  port: 8123
+  nativePort: 9000
+  enabled: false
+  persistence:
+    enabled: false
+    dataSize: 10Gi
+    logSize: 5Gi 
+
+otel:
+  port: 13133
+  nativePort: 24225
+  grpcPort: 4317
+  httpPort: 4318
+  healthPort: 8888
+  enabled: false
+
+images:
+  hdx:
+    repository: hyperdx/app
+    tag: dev
+    pullPolicy: IfNotPresent
```

Comment:
Local build of combined app+api

Will update to release repo when we are further along with v2. Mostly here for testing.